src/ralph/admin/autocomplete.py

```diff
@@ -35,7 +35,9 @@ def get(self, request, *args, **kwargs):
         """
         Returns serialized dict as JSON object.
         """
-        can_edit = ralph_site._registry[self.model].has_change_permission(
+        can_edit = not getattr(
```

Comment:
use 3-argument getattr (this will raise `AttributeError` when model has no attribute)